### DISCUSSION TITLE
classname for theming calendar property in style.css?

### DISCUSSION THREAD

**Discussion by thelazt16** [Alexays/Waybar#3338] [Q&A]
I know I can do some styling inside the config but it pretty limited and quite annoying to do. I wanted to know if I can style the calendar tooltip inside style.css instead. 
I can't find a way to make the debug mode (`GTK_DEBUG=interactive waybar`) to work with the tooltip since I can't use the element picker and it doesn't seems to show up in the objects tab nor the CSS node dropdown.

**Comment by Soulsuke** [comment]
I was about to ask the very same thing. I'm not even able to add a custom id or class elements to the spans in the calendar to style them accordingly...
